src/main/java/graphql/schema/idl/SchemaPrinter.java

```diff
@@ -874,7 +875,9 @@ private void printMultiLineDescription(PrintWriter out, String prefix, List<Stri
     }
 
     private void printSingleLineDescription(PrintWriter out, String prefix, String s) {
-        out.printf("%s\"%s\"\n", prefix, s);
+        // See: https://github.com/graphql/graphql-spec/issues/148
+        String comment = StringEscapeUtils.escapeJson(s);
```

Comment:
I am sure we have a JSON string encoder somewhere in the code base

Not sure off hand but I recall we do.